.github/workflows/ruby_tests.yml

```diff
@@ -13,10 +13,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 2.6
-      - name: Setup
-        run: |
-          gem install bundler
-          bundle install --jobs=3 --retry=3
+          bundler-cache: true
```

Comment:
`bundler-cache` is installing it in the vendor subfolder, so we would need to exclude it from Rubocop.

I'd be for not enabling it for RuboCop in this PR.
We don't need the caching so prudently here (especially once we don't know if we can have two gem caches for project). Ideally RuboCop job should install just the RuboCop strict dependencies, that should be quite minimal.

Reply:
Still, ignoring `vendor/bundle` is a good thing. There are other benefits to using `ruby/setup-ruby` instead of `actions/setup-ruby`, like that it's maintained. Having the action also perform the install means they will ensure it's set up correctly. Caching is IMHO just a nice side effect.

Reply:
Also updated. Now `BUNDLER_WITHOUT` is a global variable which means the cache of the rubocop step should also be used in the actual test run.

Reply:
Oh, of course that doesn't work. Different git repos.

Reply:
Yeah, it is creating the Gemfile.lock prior to fetching the cache, so it needs to match and those two jobs don't match for obvious resons, but it works for both great!